api/envoy/config/core/v3/protocol.proto

```diff
@@ -313,8 +313,8 @@ message Http2ProtocolOptions {
   // of WINDOW_UPDATE frames received over the lifetime of connection exceeds the value calculated
```

Comment:
Please update the comment section, since it describes the old formula.

Reply:
done

Reply:
done